src/Trigger/RabbitMQListener.cs

```diff
@@ -30,14 +36,22 @@ internal sealed class RabbitMQListener : IListener
         private bool _disposed;
         private bool _started;
 
-        public RabbitMQListener(ITriggeredFunctionExecutor executor, IRabbitMQService service, string queueName, ushort batchNumber, ILogger logger)
+        public RabbitMQListener(
+            ITriggeredFunctionExecutor executor,
+            IRabbitMQService service,
+            string queueName,
+            ushort batchNumber,

```

Comment:
remove batchNumber as well since not used.

Also, in StartAsync, use `_rabbitMQModel.BasicQos(0, 30, false);` with a TODO comment that these prefetch params should be part of RabbitMQOptions (PrefetchSize, PrefetchCount). If you have time you might just make those changes, taking the options as a param to your listener and using the values from there